### DISCUSSION TITLE
Monitor command stops after a few minutes

### DISCUSSION THREAD

**Discussion by ZacharieALES** [abraunegg/onedrive#1553] [Q&A] [answered]
I am using version v2.4.6.7 on Ubuntu.

The command onedrive --monitor stops after a few minutes. Isn't it supposed to run indefinitely and keep synchronizing?

Is there a way to prevent it from stopping?

PS: Sorry for posting this in the issues rather than in a discussion!

**Comment by abraunegg** [comment, marked as answer]
@ZacharieALES 
First issue is this: v2.4.6.7 on Ubuntu

You are running unsupported software. You must upgrade your client. Only the latest version is supported.

Once that is sorted, you are probably running into this issue: https://github.com/abraunegg/onedrive/blob/master/docs/known-issues.md#application-stops-running-without-any-visible-reason

You need to diagnose your system and your environment.

**Comment by Drachenlaeufer** [comment]
Hi!

Besides the outdated version, I see, that this issue is faced by several users in the last weeks. Maybe there was some change on Microsoft side, but actually I "solved" the problem by setting the monitor interval from 300 to 200. It could be, that you may need a different value. Test it.

**Reply by ZacharieALES** [reply]
Updating the client did not alter the behavior but modifying the interval to 200 seems to work. I have no idea why but thanks!

**Reply by Drachenlaeufer** [reply]
Just for information:
With version 2.4.13 (installed on July 14th.) I get no timeouts any more, even with monitor interval set to default 300s.
Maybe it has something to do with an update on OS or Microsoft side ... I don't know.
For the records: I use Debian.

**Comment by abraunegg** [comment]
@ZacharieALES , @Drachenlaeufer 
> Updating the client did not alter the behavior but modifying the interval to 200 seems to work. I have no idea why but thanks!

This just confirms that this is something outside of the application that is causing this.

When ever something like this is popping up here are the common elements:
1. Users are usually running Ubuntu of some sort, or a Ubuntu variant
2. Users are based in Europe, accessing the Eastern European Microsoft Data Centre


Most likely:
* Something on / within your Ubuntu Linux OS
  * Does the issue exists when not using Ubuntu?
* Your Internet Router .. plausible, but doubtful this is the cause
* Your ISP .. plausible, but doubtful
* Your network path between you and Microsoft .. plausible, but then the reports would be everywhere of an issue 
* A change at Microsoft .. plausible, but then significantly more people would be impacted

The *most* logical explanation here is that Ubuntu - or something you have installed on Ubuntu is causing this - some sort of user configuration ... 

There was also another issue where the user installed 'ufw' - Ubuntu's Firewall system ... and this caused all sorts of access issues for Docker. It would not surprise me that this is a potential culprit for this issue as well.
